### ISSUE TITLE
mWeb / Safari - Chat View Shrinks When SoftKeyboard Is Opened While Scrolling

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#13492]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1.  Click On Chat
2.  Open Soft Keyboard
3.  Scroll Through The Chat

## Expected Result: 
Chat View Should Behave Normal

## Actual Result: 
Chat View Shrinks While Scrolling

## Workaround:
unknown

## Platform:

Where is this issue occurring?

- Mobile Web Safari

**Version Number:** 1.2.37-1
**Reproducible in staging?:** y
**Reproducible in production?:** y
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/43996225/206814873-e2c59b7d-7a04-4310-8af5-7e9f9b2ebd70.MP4

https://user-images.githubusercontent.com/43996225/206814891-7e513592-a5ff-4ea0-8e35-bc3ab81763bb.mov

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @syedsaroshfarrukhdot 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1670590388576729 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by Someone-here** [comment]
This is reproducible on ios16.1 safari

**Comment by stephanieelliott** [comment]
Yep, I too was able to reproduce it. This one seems like it can be fixed externally.

**Comment by ahmdshrif** [comment]
I think it's the same issue here (same root cause).
https://github.com/Expensify/App/issues/8731

**Comment by priyeshshah11** [comment]
This issue seems to be a weird problem with iOS Safari itself not specifically Expensify. When the keyboard is opened, it actually pushes the main view up and it weirdly adds an extra scrollbar to allow users to scroll to the top of the main view. There is no clear solution to this at this stage and it happens with most websites.

**Comment by stephanieelliott** [comment]
Got it, thanks for the explanation @priyeshshah11. Given that context and the fact that we closed https://github.com/Expensify/App/issues/8731 due to it being a known bug that can only be solved with a complex workaround, I'd suggest we just close this one. Any objection, @Luke9389?

**Comment by ahmdshrif** [comment]
I think we can keep it open maybe  Apple provided solutions. 
also, why not go with a workaround since it's affected us in many cases, and for now no ideal solution!
for example, I see google and apple products are not affected by this issue. 

What do you think?

**Comment by priyeshshah11** [comment]
> and apple products are not affected by this issue. so they solve it somehow

@ahmdshrif Apple products are affected, the video attached above is apple. I think workaround can work if it doesn't have any negative side effects technically or in user experience but it's up to the team.

**Comment by ahmdshrif** [comment]
I mean apple websites . or google search, Gmail..etc. on safari ios.

**Comment by priyeshshah11** [comment]
> I mean apple websites . or google search, Gmail..etc. on safari ios. I remember I tests them on the old issue. and they work well. we can retest if you don't agree on this point.

Oh ok, got what you mean 👍 but I did test gmail & apple.com and they both should the same behaviour as this issue.

**Comment by ahmdshrif** [comment]
I am not sure what steps you follow because there are many repro steps like what I reported before https://expensify.slack.com/archives/C01GTK53T8Q/p1655052706729669 or steps on an old issue. 

here is my test on google search and I can't repro it.  

https://user-images.githubusercontent.com/21364901/207885913-ecc8a223-57ed-499a-84cb-fdd6f5b0d0e9.mp4

**Comment by priyeshshah11** [comment]
> here is my test on google search and I can't repro it.

Google search looks like a different case, because of that recommendations list. Try going to a form, I tried it by going to checkout on apple.com and I could see the extra whitespace at the bottom.

Having said that, if you think you have a possible workaround/solution then feel free to go ahead and propose it.

**Comment by ahmdshrif** [comment]
> Having said that, if you think you have a possible workaround/solution then feel free to go ahead and propose it.

not yet. but I just say we can keep it open if other websites solve it. if not we should close I agree.

**Comment by Luke9389** [comment]
@stephanieelliott I agree, I think we should close it.
